include/mbedtls/timing.h

```diff
@@ -52,7 +52,7 @@ struct mbedtls_timing_hr_time
  */
 typedef struct mbedtls_timing_delay_context
 {
-    struct mbedtls_timing_hr_time   timer;
+    struct mbedtls_timing_hr_time   MBEDTLS_PRIVATE(timer);
```

Comment:
This was missed by python script.
I checked the doxygen xml files: Doxygen claims this variable is defined at line 47. I don't know why this happens. However, since all other 'missed' variables were covered by adjusting library configuration, I don't see any value in looking into this any further.